src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -520,9 +522,21 @@ Namespace Microsoft.CodeAnalysis.Semantics
         End Function
 
         Private Function CreateBoundConversionOperation(boundConversion As BoundConversion) As IConversionExpression
-            Dim operand As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundConversion.Operand))
+            Dim operand As Lazy(Of IOperation)
+            Dim methodSymbol As MethodSymbol
+
+            ' TODO: Use Gen's GetConversion after https://github.com/dotnet/roslyn/pull/21445 is merged.
```

Comment:
When https://github.com/dotnet/roslyn/pull/21445 is merged, I'll update this PR and remove this code.